taxi/urls.py

```diff
@@ -46,6 +51,31 @@
     path(
         "drivers/<int:pk>/", DriverDetailView.as_view(), name="driver-detail"
     ),
+    path(
+        "drivers/create/",
+        DriverCreateView.as_view(),
+        name="driver-create"
+    ),
+    path(
+        "drivers/<int:pk>/delete/",
+        DriverDeleteView.as_view(),
+        name="driver-delete"
+    ),
+    path(
+        "license/<int:pk>/update/",
+        DriverLicenseUpdate.as_view(),
+        name="driver-update"
+    ),
+    path(
+        "drivers/car/<int:car_pk>/<int:driver_pk>/update/",
+        driver_car_update,
+        name="assign-car"
+    ),
+    path(
+        "drivers/car/<int:car_pk>/<int:driver_pk>/delete/",
+        driver_car_delete,
+        name="reassign-car"
```

Comment:
why do u need 2 pk here? U can use only one here
also, it seems to me, that url should start from car/car_pk/f.e. update driver 
cause u are using this views from car-detail page